finance/expenses/Portfile

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/manojkarthick/expenses 0.2.2 v
+categories          finance
+maintainers         {gmail.com:smanojkarthick @manojkarthick} \
+                    openmaintainer
+license             MIT
+
+description         Simple, Interactive, command line Expense logger.
+
+long_description    An interactive command line expense logger. \
+                    Answer a series of questions to log your expenses. \
+                    Currently writes to CSV and SQLite backends.
+
+checksums           ${distname}${extract.suffix} \
+                        rmd160  daf975700c0b1aa8a97008d5950425884fc5ae2a \
+                        sha256  e7e1d309a2b2aa83948917532bfdd8dfd8eebf1ce1b5a04ee3b4d14b77c41f5e \
+                        size    35912
+
+build.pre_args-append    -ldflags \
+    '-X github.com/manojkarthick/expenses/cmd.Version=${version}'
```

Comment:
nitpick: Just to neaten up the spacing here, I don't think `-append` is required.